.github/ISSUE_TEMPLATE/new_pod_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: New POD request
+about: Request to add a new pod to the MDTF-diagnostics repo
+labels: 'enhancement'
+---
+
+**Describe the science question your POD addresses**
+
+**Describe the analysis your POD will perform**
+[e.g., my POD computes the 3-month average of wind speed over DJF,MAM, JJA, SON from 1970--2020]
+
+**What output does your POD generate?**
+[e.g., my POD produces a 4-panel figure of seasonal composite maps of 850-hPa wind speed ]
+
+**Describe any required input files**
```

Comment:
It would be good to split this into two:  what model output is required and what observational data is needed